### ISSUE TITLE
As a custom rule author, I want to see my custom attributes in the engine report

### ISSUE THREAD

**Issue by nhaydel** [cdisc-org/cdisc-rules-engine#469]
Acceptance Criteria:

1. A user has a way of defining how custom attributes appear in the report.
2. Based on this definition, the engine should populate the custom attributes in the report on the Rules Report tab.

**Comment by SFJohnson24** [comment]
@rradel we discussed this in scrum meeting today--we wanted your input.  We could put all custom attributes from the rules onto the rules report or we could create a CLI flag that lets you choose which attributes you would want put into the report.  The report would have that category in the report or be blank if it is not there.  It would also have an 'ALL' command where it iterates through the custom category and reports all the categories in the report.  We wanted your input since you had requested this feature in #741

**Comment by rradel** [comment]
@SFJohnson24 I agree with the direction you're proposing. Having the category available in the YAML rule itself is definitely the key piece for our use case, as it allows us to filter/select rules based on specific purposes for validation. Including those attributes in the report is less critical for us, but I can see how it could be valuable for broader reporting or debugging use cases. I like the CLI flag approach. It offers flexibility without cluttering the report by default

**Comment by SFJohnson24** [comment]
https://cdisc-org.github.io/conformance-rules-editor/#/customrules  I think a nice way of doing this would be adding something in output variables where custom attributes could be selected and then reported in the report.  I do think the optionality of the CLI flag is good but I think the ability to specify in your rule that you want attribute x y z in your rule output would also be good
